Source/WebCore/dom/TrustedType.cpp

```diff
@@ -173,6 +187,33 @@ ExceptionOr<String> trustedTypeCompliantString(TrustedType expectedType, ScriptE
     return stringValue;
 }
 
+String getTrustedTypeForAttribute(const String& tagName, const String& attributeParameter, const String& elementNamespace, const String& attributeNamespace)
```

Comment:
"get" prefix should only be used for a function with an out argument.
This should be named trustedTypeForAttribute instead.